src/main/java/core/basesyntax/dao/impl/MessageDaoImpl.java

```diff
@@ -12,21 +14,64 @@ public MessageDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public Message create(Message entity) {
-        return null;
+        Session session = null;
+        Transaction transaction = null;
+        try {
+            session = factory.openSession();
+            transaction = session.beginTransaction();
+            session.persist(entity);
```

Comment:
let's keep our code consistent, use either `save()` or `persist()` everywhere and corresponding cascade type where needed

Reply:
i don't clearly understand
where we have not cascade i use save()
where we have to use cascade, I use methods that equals cascade's types
Do u say just change my save methods?